autoload/jedi.vim

```diff
@@ -68,8 +68,7 @@ function! s:init_python() abort
     endif
 
     " Handle "auto" version.
-    if has('nvim') || (has('python') && has('python3'))
-        " Neovim usually has both python providers. Skipping the `has` check
+    if (has('python') && has('python3'))
```

Comment:
I think we could improve it here, but I would then expect `let g:loaded_python_provider = 1` for it to be available.
So in this case you should set `let g:loaded_python_provider = 0`, which also would skip loading it, but then can be considered to mean "do not use it".